vars/buildPluginWithGradle.txt

```diff
@@ -0,0 +1,19 @@
+<p>
```

Comment:
Now that you updated the documentation in the README, the below paragraphs should also be updated accordingly. E.g. just reference the README in "some key features are not supported: ..." and "see the Pipeline code".